app/helpers/application_helper.rb

```diff
@@ -185,4 +185,12 @@ def logo_path
       image_url("navbar-logo-beta.jpg")
     end
   end
+
+  def show_beta_banner?
```

Comment:
I think we should whitelist rather than blacklist. I don't feel like I know every controller in the system and don't feel super comfortable putting the banner on all the pages.

Can we just pick up DashboardController, InboxController, GroupsController and DiscussionsController?

Also whenever you have an if condition where the body of the condition is true or false, you can safely remove the condition and just evaluate the statement.

So in this case I think the method should be:
```
['dashboard_controller', 'groups_controller', 'discussions_controller'].include? controller_name
```